react/features/participants-pane/components/breakout-rooms/components/native/CollapsibleRoom.js

```diff
@@ -46,28 +45,28 @@ export const CollapsibleRoom = ({ room, searchString }: Props) => {
         = `${room.name
     || t('breakoutRooms.mainRoom')} (${roomParticipantsNr})`;
 
-    // Regarding the fact that we have 3 sections, we apply
-    // a certain height percentage for every section in order for all to fit
-    // inside the participants pane container
-    const containerStyle
-        = roomParticipantsNr > 2 && styles.collapsibleRoomContainer;
-
     return (
         <CollapsibleList
-            containerStyle = { containerStyle }
             onLongPress = { _openContextMenu }
             title = { title }>
             <FlatList
                 bounces = { false }
                 data = { Object.values(room.participants || {}) }
                 horizontal = { false }
                 keyExtractor = { _keyExtractor }
-                // eslint-disable-next-line react/jsx-no-bind
+
+                // For FlatList as a nested list of any other FlatList or SectionList
+                // we have to pass a unique value to this prop
+
+                /* eslint-disable-next-line react/jsx-no-bind */
+                listKey = { (item, index) => `key_${index}` }
```

Comment:
This is the only way I managed to escape:

[Invariant Violation: A VirtualizedList contains a cell which itself contains more than one VirtualizedList]

Everytime I Add Breakout Room I get this warning if I don't use listKey as it is used above.